nosso-calendario/src/main/java/com/br/devefc/nossocalendario/rest/AgendaForm.java

```diff
@@ -0,0 +1,32 @@
+package com.br.devefc.nossocalendario.rest;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import com.br.devefc.nossocalendario.model.Agenda;
+import com.br.devefc.nossocalendario.model.User;
+
+public class AgendaForm {
+
+	@NotNull(message = " O nome não pode ser nulo")

```

Comment:
*Usamos tudo que conhecemos que está pronto. Só fazemos código do zero se for estritamente necessário.*
Aqui da para usar @NotBlank. Ele já pega os dois :).